src/tests/Common/XUnitWrapperGenerator/XUnitWrapperGenerator.csproj

```diff
@@ -5,6 +5,7 @@
     <Nullable>enable</Nullable>
     <EnableDefaultItems>true</EnableDefaultItems>
     <IsRoslynComponent>true</IsRoslynComponent>
+    <PublishTrimmed>false</PublishTrimmed>
```

Comment:
Given that `PublishTrimmed` behavior is really only defined for applications, would it instead make sense to modify https://github.com/dotnet/runtime/blob/9eb026696ab3f2e5f486838fe3800679885ba849/src/tests/Directory.Build.props#L210 to only enable trimming for executables?

Reply:
That seems like a better way of solving the problem. I'm not very familiar with this part of the build process though so I'm not sure what the implications of that would be. cc @mdh1418

Reply:
The property `PublishTrimmed` set by default for the runtime tests may improve AOT compilation time by excluding unnecessary assemblies. However, according to the comment below it is not intended behavior.

> PublishTrimmed is expected to be set on per-project basis, not globally for entire subtrees.

In order to avoid additional complexity in the `MergedRunner` and set `PublishTrimmed` on per-project basis, we can remove these props.
https://github.com/dotnet/runtime/blob/9eb026696ab3f2e5f486838fe3800679885ba849/src/tests/Directory.Build.props#L208-L212

Note: The `XUnitWrapperGenerator` project may become obsolete soon, as Xunit wrappers are not used for the Mono runtime tests targeting apple mobile platforms due to parallel AOT compilation on Helix. We plan to do the same for the Native AOT runtime tests.

Reply:
@simonrozsival You may change the property group condition below to include `Condition="'$(TargetsAppleMobile)' == 'true' and '$([System.String]::Copy($(OutputType)).ToLower())' == 'exe'"`.

https://github.com/dotnet/runtime/blob/9eb026696ab3f2e5f486838fe3800679885ba849/src/tests/Directory.Build.props#L208